docs/tests/audit/test_rpc_calls.md

```diff
@@ -4,75 +4,84 @@ Tests for rpc endpoints (check availability and basic functionality)
+| TestRpcGetTransaction::test_get_transaction_receipt                                 | Check response structure for getTransactionReceipt (eth & neon methods)         |           |
+| TestRpcGetTransaction::test_get_transaction_receipt_when_hash_doesnt_exist          | Check getTransactionReceipt if tx hash not exist (eth & neon methods)           |           |
+| TestRpcGetTransaction::test_eth_get_transaction_receipt_when_hash_doesnt_exist      | Non existent hash as param                                                      |           |
+| TestRpcGetTransaction::test_neon_get_transaction_by_sender_nonce_negative           | Negative cases with posible error checkes                                       |           |
+| TestRpcGetTransaction::test_neon_get_transaction_by_sender_nonce_plus_one           | Request nonce+1, which is not exist                                             |           |
+| TestRpcGetTransaction::test_neon_get_transaction_by_sender_nonce                    | Positive cases                                                                  |           |
+|                                                                                     |                                                                                 |           |
+| TestRpcEstimateGas::test_eth_estimate_gas                                           | Get estimate gas for contract                                                   | NDEV-2310 |
+| TestRpcEstimateGas::test_eth_estimate_gas_negative                                  | Get estimate gas without params                                                 | NDEV-2310 |
+| TestRpcEstimateGas::test_eth_estimate_gas_with_big_int                              | Get estimate gas for a big contract                                             |           |
+| TestRpcEstimateGas::test_rpc_estimate_gas_send_neon                                 | Get estimate gas for send neon transfer operation                               |           |
+| TestRpcEstimateGas::test_rpc_estimate_gas_erc20                                     | Get estimate gas for erc20 transfer operation                                   |           |
+| TestRpcEstimateGas::test_rpc_estimate_gas_spl                                       | Get estimate gas for spl transfer operation                                     |           |
+| TestRpcEstimateGas::test_rpc_estimate_gas_contract_get_value                        | Get estimate gas for getting value from the contract                            |           |
+| TestRpcEstimateGas::test_rpc_estimate_gas_contract_set_value                        | Get estimate gas for setting value in the contract                              |           |
+| TestRpcEstimateGas::test_rpc_estimate_gas_contract_calls_another_contract           | Get estimate gas for calling function in one contract from another              |           |
```

Comment:
Class names are not the same as in test files
And structures are different

Reply:
Updated